ansible/tasks/initdb.yml

```diff
@@ -61,4 +92,17 @@
     password: "{{ db_password }}"
     force_basic_auth: yes
   with_items: "{{ db.authkeys }}"
-  when: result.status == 404
+  when: dbexists is defined and dbexists.status == 404
+
+- name: secure db against anonymous access
```

Comment:
I'll need further guidance on how to do this as part of the couchdb role in a way that allows for separate toggle on/off - since this is orthogonal to the main change in this pr, I will remove it and deal with it separately.